src/x/__tests__/xHilbertTransform.test.ts

```diff
@@ -77,3 +77,73 @@ describe('test fast hilbert transform', () => {
     }
   });
 });
+
+describe('test fast hilbert transform with forceFFT (array length greater than a power of 2)', () => {
+  const length = 2 ** 6;
+  const x = new Float64Array(length);
+  const cos = new Float64Array(length);
+  const sin = new Float64Array(length);
+  const minusCos = new Float64Array(length);
+  const t = length;
+  for (let i = 0; i < length; i++) {
+    x[i] = i;
+    cos[i] = Math.cos((i * Math.PI) / (t / 2));
+    sin[i] = Math.sin((i * Math.PI) / (t / 2));
+    minusCos[i] = -Math.cos((i * Math.PI) / (length / 2));
+  }
+  const diff = 10;
+  const scos = xSampling(cos, { length: length + diff });
+  const ssin = xSampling(sin, { length: length + diff });
+  const tcos = xSampling(xHilbertTransform(scos, { forceFFT: true }), {
+    length,
+  });
+  const tsin = xSampling(xHilbertTransform(ssin, { forceFFT: true }), {
+    length,
+  });
```

Comment:
Don't put test code in describe blocks. Describes are executed before the test phase, so you can have issues with debugging and you will execute the code even if you filter the tests/skip/watch